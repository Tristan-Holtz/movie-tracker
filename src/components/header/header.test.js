import React from 'react';
import { Header } from './Header';
import { shallow } from 'enzyme';

describe('header', () => {
  it('should match a snapshot', () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot();
  })
})