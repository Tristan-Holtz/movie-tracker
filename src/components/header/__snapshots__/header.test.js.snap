// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header should match a snapshot 1`] = `
<header>
  <h1>
    Movie Tracker
  </h1>
</header>
`;
